### ISSUE TITLE
Connections are not being displayed

### ISSUE THREAD

**Issue by zamlz** [org-roam/org-roam-server#154]
![2021-05-31_12:13:48_993x744](https://user-images.githubusercontent.com/10750027/120233085-d3de7d80-c209-11eb-882e-beaee1faeca2.png)

**Ouput of `org-roam-diagnostics`:**

```
Copy info below this line into issue:
- Emacs: GNU Emacs 27.2 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.27, cairo version 1.17.4)
 of 2021-03-26
- Framework: N/A
- Org: Org mode version 9.4.6 (9.4.6-ga0a87d @ /home/zamlz/.config/emacs/straight/build/org/)
- Org-roam: 1.2.3
```

Standard Backlinks still appears to work correctly when it is run in a `org-roam` buffer.

I was working fine yesterday, but then I did a package update today morning. I'm not sure what could have broken it.

**Comment by goktug97** [comment]
Can you rebuild your database. Remove org-roam.db to build it from scratch. If doesn't work can you try with this commit: ea4082f54af14cb692d6eb7f8f085971ebdca69d

**Comment by zamlz** [comment]
> Can you rebuild your database. Remove org-roam.db to build it from scratch. If doesn't work can you try with this commit: [ea4082f](https://github.com/org-roam/org-roam-server/commit/ea4082f54af14cb692d6eb7f8f085971ebdca69d) because recently there was a PR #149 and I had the same problem with it but before merging to the master it is fixed.

For sure, so I already tried rebuilding the database after deleting the file but no luck there.

But tihs commit should fix it I'm guessing? If it's merged to master, I wonder if my `straight.el` has pulled it in already in that case. I'll go through this and see if it fixes it.

**Comment by goktug97** [comment]
No no, that commit is before the PR. I just wanted to see whether the PR broke your setup or not.

**Comment by zamlz** [comment]
ah that makes sense. I managed to figure it out though by reading through your comments in PR #149. It appears that the clue was in the webdev console on firefox. It let me know that a variable that I've set was not correct.

I guess for whatever reason this line of code worked before
```lisp
(setq org-roam-server-network-arrows t)
```

But it looks like the spec changed now and I have to provide a type, `to`, `from`, etc. For now, I just put down `nil`.
```lisp
(setq org-roam-server-network-arrows nil)
```
